components/txn_types/src/write.rs

```diff
@@ -217,9 +291,25 @@ impl WriteRef<'_> {
         if self.has_overlapped_rollback {
             b.push(FLAG_OVERLAPPED_ROLLBACK);
         }
+        if let Some(ts) = self.gc_fence {
+            b.push(GC_FENCE_PREFIX);
+            b.encode_u64(ts.into_inner()).unwrap();
```

Comment:
Why not `encode_var_u64`?

Reply:
VarInt saves space when ts is 0. But VarInt occupies more space (9 bytes vs 8 bytes) and is slower when ts is a real timestamp.
It has little difference to me because overlapping happens rarely...

Reply:
I thought it doesn't matter much which encoding to use here, but encode_u64 looks faster than var_u64

Reply:
But it's strange to encode start_ts in var int and this ts in u64.

Reply:
You can see in Lock's encoding, everything new is using encode_u64...